tools/sistr_cmd/sistr_cmd.xml

```diff
@@ -1,9 +1,9 @@
-<tool id="sistr_cmd" name="sistr_cmd" version="1.0.2">
+<tool id="sistr_cmd" name="sistr_cmd" version="1.1.1">
```

Comment:
Since the version number here is defined in 2 places, one thing you could do is make it a variable. This can be accomplished by adding a section like below to the XML.

```xml
    <macros>
        <token name="@VERSION@">1.1.1</token>
    </macros>
```

Then, you can replace any usage of `1.1.1` with `@VERSION@` elsewhere in this file.

As an example you can look at staramr: https://github.com/phac-nml/galaxy_tools/blob/master/tools/staramr/staramr_search.xml#L1. The version string is `@VERSION@+galaxy2` here, with `galaxy2` used to specify a non-trivial update to the Galaxy wrapper that doesn't update the underlying tool (you don't need the `+galaxy` part right now, but if we updated this wrapper again that's how we'd increment the version).

Reply:
Nice `marcos` feature use. I like variable option for easier future maintenance. Thank you for this hint. I've adopted it in the new commit